src/config-import.js

```diff
@@ -133,29 +144,29 @@ export async function readConfig(configPath) {
     },
 
     /**
-     * @returns {Iterable<{ fieldNames: string[], iconName: string | undefined, value: import('@mapeo/schema').PresetValue }>}
+     * @returns {Iterable<{ fieldNames: string[], iconName: string | undefined, value: import('@mapeo/schema').PresetValue, name: String}>}
```

Comment:
Nitpick (here and elsewhere in the file): could we use `string` instead of `String`? I _think_ TypeScript says these are equivalent but I really don't wanna use the [`String` object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#string_primitives_and_string_objects) by accident.